COSC481W-2024Fall/MLite
# Added S3 setup for secure production storage

Features:

- Set up S3 storage specifically for production to securely store datasets and models
- Implement access control policies and encryption for secure and reliable data handling
- Integrated S3 with the production server to enable seamless loading of datasets and models

Testing:

- Deploy to production environment
- Verify that the production server can upload, access, and load datasets/models from S3
- Confirm access control and encryption policies are enforced on S3
- Test multiple scenarios to ensure smooth, secure data retrieval in production

Issue: #23
